tests/Messages/BatchMessageTest.php

```diff
@@ -41,6 +41,20 @@ public function testAddRecipient()
         $this->assertEquals(1, $array['recipients']['to']);
     }
 
+    public function testAddRecipientWithoutFirstAndLastName()
+    {
+        $message = $this->client->BatchMessage($this->sampleDomain);
+        $message->addToRecipient('test@samples.mailgun.org', 'this-is-not-an-array');
+        $messageObj = $message->getMessage();
+        $this->assertEquals(['to' => ['test@samples.mailgun.org']], $messageObj);
+
+        $reflectionClass = new \ReflectionClass(get_class($message));
+        $property = $reflectionClass->getProperty('counters');
+        $property->setAccessible(true);
+        $array = $property->getValue($message);
```

Comment:
👍 maybe we can use it when new v3 release is ready. For now I chose to use PHPUnit because all tests are using this package. Anyway, thanks for the idea!